packages/angular/cli/utilities/config.ts

```diff
@@ -151,7 +151,7 @@ export function getProjectByCwd(workspace: experimental.workspace.Workspace): st
   }
 }
 
-export function getPackageManager(): string {
+export function getPackageManager(): string | null {
```

Comment:
Maybe this should change to `getConfiguredPackageManager ` to avoid confusion?

Reply:
Done.